### ISSUE TITLE
API to retrieve personal information.

### ISSUE THREAD

**Issue by busi-reddy-karnati** [WazirX/wazirx-api#6]
Can we have an API that gives personal information like
1. Different Holdings of coins
2. Order Statuses and History
3. Place orders using API
4. Transactions executed to and from the bank account

I believe that this will be a great value addition to the application and I can help with the code for the same, if needed.

Thank you.
Looking forward to hearing from you.

**Comment by hkmekala** [comment]
I would be happy to move from cex.io to here if the team provides an API to place orders, withdraw funds also it would be helpful to have an estimated transaction fee details for high-value coins which can make a trade not worth it just because of the transaction fee.

**Comment by jackzbazuka** [comment]
yeah , it'd be really helpful to track personal information off-platform
